snmp/ups-apcups.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+################################################################
+# copy this script to somewhere like /opt and make chmod +x it #
+# edit your snmpd.conf and include                             #
+# extend ups-apcups /opt/ups-apcups.sh                         #
```

Comment:
We need to standardise the install directory - people can change it but we should tell those that don't know to / want to. I think we have suggestions in the docs of where to store these, use one of those.